### PR TITLE
Fixes unicode rendering in Admin>Secrets>Admin Log/Show Admins

### PR DESCRIPTION
**Pull request by Low3**
## About The Pull Request
Admin Log and Show Admins now correctly show unicode characters instead of corrupted symbols.

## Why It's Good For The Game
Slightly better admin experience.

## Changelog
:cl:
fix: Admin Log and Show Admins in Secrets panel now render unicode characters.
/:cl:

### CODE REVIEW

**Review by Rohesie — APPROVED** [review]

File: code/modules/admin/verbs/secrets.dm [hunk 1]
```diff
@@ -53,14 +53,14 @@
 	switch(action)
 		//Generic Buttons anyone can use.
 		if("admin_log")
-			var/dat = "<B>Admin Log<HR></B>"
+			var/dat = "<meta charset=UTF-8><B>Admin Log<HR></B>"
 			for(var/l in GLOB.admin_log)
 				dat += "<li>[l]</li>"
 			if(!GLOB.admin_log.len)
 				dat += "No-one has done anything this round!"
 			holder << browse(dat, "window=admin_log")
```
Comment (AnturK) [inline code/modules/admin/verbs/secrets.dm:61]: This PR is fine but the fact we have direct browse calls in tgui act handling is depressing.

File: code/modules/admin/verbs/secrets.dm [hunk 2]
```diff
@@ -53,14 +53,14 @@
 	switch(action)
 		//Generic Buttons anyone can use.
 		if("admin_log")
-			var/dat = "<B>Admin Log<HR></B>"
+			var/dat = "<meta charset=UTF-8><B>Admin Log<HR></B>"
```
Comment (AnturK) [inline code/modules/admin/verbs/secrets.dm:56]: Could you wrap the charset in quotes for my sanity though ?

**Review by AnturK — CHANGES_REQUESTED** [review]

File: code/modules/admin/verbs/secrets.dm (hunk 2 above)
Comment (Low3) [reply code/modules/admin/verbs/secrets.dm:56]: It doesn't seem to work this way

Comment (Low3) [reply code/modules/admin/verbs/secrets.dm]: Done

File: code/modules/admin/verbs/secrets.dm [hunk 3]
```diff
@@ -53,14 +53,14 @@
...
-			var/dat = "<B>Admin Log<HR></B>"
+			var/dat = "<meta charset='UTF-8'><B>Admin Log<HR></B>"
 			for(var/l in GLOB.admin_log)
 				dat += "<li>[l]</li>"
 			if(!GLOB.admin_log.len)
 				dat += "No-one has done anything this round!"
 			holder << browse(dat, "window=admin_log")
 		if("show_admins")
-			var/dat = "<B>Current admins:</B><HR>"
+			var/dat = "<meta charset=UTF-8><B>Current admins:</B><HR>"
```
Comment (AnturK) [inline code/modules/admin/verbs/secrets.dm:63]:
```suggestion
			var/dat = "<meta charset='UTF-8'><B>Current admins:</B><HR>"
```
Here too please.

Comment (Low3) [reply code/modules/admin/verbs/secrets.dm]: Right, forgot about that one. Done.

**Pull request merged**